### ISSUE TITLE
Add a flag button/API to put up a product for review when you're in a hurry

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#1408]
![image](https://user-images.githubusercontent.com/1689815/46919128-f8df3d00-cfda-11e8-9f2b-cfc0ef94483e.png)

**Comment by teolemon** [comment]
This should only be for admins. Otherwise, we decrease the incentive for users to fix the issue themselves.

**Comment by teolemon** [comment]
A notification would be sent to the moderation channel

**Comment by teolemon** [comment]
Would probably write in a dedicated quality facet

**Comment by stephanegigandet** [comment]
We need at least a text field to state what is wrong with the product. Whenever there's a notice on Slack "Look at this product: ... " without any information, I almost never see what is supposed to be wrong. There's no point having multiple people trying to guess what the reporter noticed.

**Comment by CharlesNepote** [comment]
@stephanegigandet I agree!! See the review spreadsheet we are testing with @teolemon thanks to my Power User extension: https://docs.google.com/spreadsheets/d/1DE85Or0QiYwIXcG4vSVZyFSLMKvmJqOXM5ooJzxZr6Y/ (you will have to ask for access).

**Comment by teolemon** [comment]
We should have thumb up and down on all revisions. That way we'll eventually be able to flag problematic edits
